CMakeLists.txt

```diff
@@ -335,9 +335,9 @@ if(SINGULARITY_USE_SPINER)
 endif()
 
 if(SINGULARITY_USE_KOKKOS)
-  singularity_enable_kokkos(singularity-eos_Interface)
+  singularity_enable_kokkos(singularity-eos_Common)
```

Comment:
Maybe worth commenting that this is to make cray happy.

Reply:
I'm not 100% that's true to be honest. I'll try the same process on a non-cray machine and see if I get the same result.

We weren't able to reproduce on non-cray machines but that wasn't with the EAP build process.

Reply:
Yeah, I'm going to push back on this comment. The Fortran interface test _is_ dependent on Kokkos when we build with Kokkos, and I don't think that's covered by the `*Interface` library. I think that's why it needs to be linked to the `*Common` library.

Reply:
Ah ok. Maybe add a comment to that effect? I'm mostly concerned about someone changing it back, since this change is non-obvious.

Reply:
LGTM